.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           fetch-depth: 0
 
       - name: ci/setup-go
-        uses: actions/setup-go@6edd4406fa81c3da01a34fa6f6343087c207a568 # v3.5.0
+        uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # v3.5.0
```

Comment:
Can we get this version number updated to avoid possible future confusion? Same thing for the one below.